### ISSUE TITLE
Tweak brand manual for release, remove draft, improve font spacing etc etc

### ISSUE THREAD

**Issue by parthlawate** [joomla/joomla-marketing#46]
This is for @Siddharth

**Comment by siddharthkabra** [comment]
@parthlawate you can expect some outputs on the tweak soon

**Comment by siddharthkabra** [comment]
![image](https://cloud.githubusercontent.com/assets/25842720/23748948/cc318756-04eb-11e7-9d7a-d7f4b6db75e0.png)

**Comment by siddharthkabra** [comment]
we saw that the current Joomla Logotype had spacing issues especially evident in small type. The 'm', 'l' and 'a' and '!' seem to get all muddled up. Just rearranged the spacing and have tweaked the letters to make it better. Another exploration was also to tweak the insignia a little for better fidelity.

**Comment by siddharthkabra** [comment]
![image](https://cloud.githubusercontent.com/assets/25842720/23749031/3743fe16-04ec-11e7-9a25-9a58cea6765c.png)

**Comment by MATsxm** [comment]
of course you know this is a registered TM?

**Comment by parthlawate** [comment]
Option 2 looks good !  About 3 might be too big a change..   @Mat Option 2 was requested as a tweak to the brand manual before we finalize it since its hard to read when in a small size

**Comment by Sandra97** [comment]
Siddardth, do not touch the logos, they are registered!

**Comment by MATsxm** [comment]
are you kiddin me?
IT IS A LEGAL ISSUE!!!
you can NOT change it!

**Comment by siddharthkabra** [comment]
@Sandra97 sure they are registered. But, one could register these as well. And its not a Legal issue. Brands change logos and tweak them all the time.

**Comment by siddharthkabra** [comment]
I think we can keep it just to the font spacing as of now

**Comment by MATsxm** [comment]
are you crazy or what? you will pay for it and impact legal issues pending?

**Comment by MATsxm** [comment]
![fae76cf48d8a300ac0ab35ee082d8ea9](https://cloud.githubusercontent.com/assets/1972717/23749169/46c35ed0-049d-11e7-9fe5-6051d7b9af7a.png)

**Comment by siddharthkabra** [comment]
Everytime a logo is made, it is registered and trade-marked. Changes to the logo can also be registered and trademarked. It is done all the time whenever companies and brands decide to tweak logos. Be sure that there is no legal issue around this. If we choose to go ahead, we will have to register the new one as well. That's all!

**Comment by MATsxm** [comment]
OH BOY!!!!
please take care about your job, this is just crazy!
what ever you pretend it does NOT work like you.

**Comment by siddharthkabra** [comment]
@MATsxm we do this all the time for brands and it's perfectly ok legally. I hope you've seen Google recently change their identity. It's alright to do it.

![image](https://cloud.githubusercontent.com/assets/25842720/23749497/703bd8a4-04ee-11e7-8354-9cd99db6795a.png)

**Comment by MATsxm** [comment]
oh boy!!!!

**Comment by Sandra97** [comment]
No discussion is needed. **You cannot change anything in the logos**.
Please go back to your inition mission:

- remove "draft" anywhere it's mention.
- increase the font-size of the content to improve lisibility. 

Nothing more.

**Comment by coolcat-creations** [comment]
-1 👎  I suggest to close this (please)

**Comment by Sandra97** [comment]
I'm closing the issue and will re-open one with the initial mission I mentioned above.

**Comment by aDaneInSpain** [comment]
Such a shame you are all shutting him down like this. The font of the logo is awful and very dated and always have been. Why can we not at least talk about improving it? Would it really be that hard?

**Comment by aDaneInSpain** [comment]
I do not think we need to change the icon, but the font for "Joomla" is squashed and looks a little amateurish IMO.

**Comment by aDaneInSpain** [comment]
Also, we DID change the icon before, when we removed the shininess and made it "flat"?

**Comment by siddharthkabra** [comment]
I think we should at least begin with improving the font and the spacing. If we can have some more constructive feedback, we can make it better. thanks @aDaneInSpain

**Comment by Sandra97** [comment]
Talking about changing it is a thing (like talking about removing the bang ;) ) but one person cannot decide to do any change on a registered logo.

**Comment by siddharthkabra** [comment]
No ones 'deciding' anything unilaterally. It's a proposal, and hence, it's put up here. We need a group effort and positive constructive feedback and iterations to see if it works.

**Comment by aDaneInSpain** [comment]
Of course that is not something one person can just do. But talking about it should not be shut down like this IMO. The improved letter spacing suggested by @siddharthkabra is a huge improvement IMO and I know that this is also something @liliespinosa feels could really do with improving. In fact (when I initially tried to get her on board as a contributing designer, she felt hesitant because the logo is so bad, in her opinion).

These things can have a major impact and if people in general feel the logo is unprofessional. It does not hurt to talk about it.

**Comment by Sandra97** [comment]
Soren, as I said in one of my previous comment, I'll open a new issue to first apply the changes we agreed in London and then we can all have a discussion of what can/should be done regarding any registered element.
First emergency is to remove 'draft' which is here since 2 years now. So, let's do step by step.

**Comment by beat** [comment]
Suggestions for improving logo and at least the font of Joomla! made by @siddharthkabra look great imho and are a significant proposal with potentially a great impact.

They could sure be seen as a different topic than this tracker item, but they would make sense eventually, e.g. specially when finances allow for the slight change to be registered. As it would only be a slight change, there is most probably no legal impact on existing litigations. Companies tweak their logo regularly, and manage litigation in parallel. ianl but as international TMs holder know a bit about it.

This is an open-source project driven by volunteers like everyone in here, and positive contributions proposals should always be received gratefully for an open-minded discussions. Same is valid in corporations, although depending on corporation culture. *Respect* should always prevail, even when we strongly disagree. So tone is hugely important, specially in writing and remotely. ; -)

**Comment by MATsxm** [comment]
@aDaneInSpain 

> Also, we DID change the icon before, when we removed the shininess and made it "flat"?

Color and shape are not part of the TM (read the patern)
Anyway this is not part of the job and ask OSM if you want.... oh wait.... who in OSM? ;)

**Comment by Radek-Suski** [comment]
Guys let's calm down:
- We can't change the logo. To be honest I don't think we even can change the spacing because it's a part of one of those logos. 
- As much I appreciate your efforts, we cannot simply change it like this. It would require participation of the Legal Department and then the OSM board of directors. I also don't think it is particularly cheap to update the trademark. We are not Google ;)

**Comment by parthlawate** [comment]
Just for the sake of understanding. I think when the brand manual was created (March 2015) logo variations were done to make it flat etc.. https://tm.joomla.org/files/docs/20150308_Joomla_Brand 

At this point we already had the mark registered - am i right ? We are trying to finalise the same brand manual and publish it.. The main reason this issue was opened was to increase the text spacing (not change the logo) 

But as a point of academic interest. If the brand manual has variations from the registered TM , will we need to register a change ?

**Comment by JoeJoomla** [comment]
I’m going from memory here but I believe it’s just the word Joomla! that is registered and not the graphic. That’s why it was easy to make the graphic flat.

**Comment by mbabker** [comment]
My two cents, I don't see the issue looking into a change.  But that should be separate from getting the brand manual published.  They were right to ask for changes to the logo to not be included in this task, it's unrelated work.  It could've been communicated better most likely, but when has Joomla ever been all that great with communication to begin with?

So if there really is a desire to go through with it, work on it through a separate tasking please.  It's not much different than keeping code issues isolated to a single bug or feature request, you don't start mixing all sorts of things together because it becomes rather complex to keep track of things and mixes responsibilities.
